store/models/common.go

```diff
@@ -292,59 +289,94 @@ func (w *WebURL) Scan(value interface{}) error {
 	return nil
 }
 
-// Time holds a common field for time.
-type Time struct {
+// AnyTime holds a common field for time, and serializes it as
+// a json number.
+type AnyTime struct {
```

Comment:
Why the name "AnyTime"?

Reply:
Because the main way this type differentiates from our other `Time` types is that it can parse a variety of JSON input, hence the word `Any`. To be more explicit, it could be `AnyInputTime`.